packages/node/src/integrations/http.ts

```diff
@@ -114,7 +114,7 @@ function _createWrappedRequestMethodFactory(
         if (parentSpan) {
           span = parentSpan.startChild({
             description: `${requestOptions.method || 'GET'} ${requestUrl}`,
-            op: 'request',
+            op: 'http',
```

Comment:
hmm, this should probably be `http.client`

Reply:
Yup, you're right. Will fix.